pandas/tests/strings/test_case_justify.py

```diff
@@ -278,6 +278,8 @@ def test_center_ljust_rjust_mixed_object():
 
 
 def test_center_ljust_rjust_fillchar(any_string_dtype):
+    if any_string_dtype == "string[pyarrow_numpy]":
+        pytest.skip("Arrow logic is different")
```

Comment:
Can you flesh out this message, possibly with just a link to https://github.com/pandas-dev/pandas/pull/54533/files#r1299808126

Reply:
Added the link